main/src/cgeo/geocaching/models/Geocache.java

```diff
@@ -1869,16 +1869,15 @@ public boolean hasStaticMap() {
     }
 
     /**
-     * Add spoilers stored locally in <tt>/sdcard/GeocachePhotos</tt>. If a cache is named GC123ABC, the
-     * directory will be <tt>/sdcard/GeocachePhotos/C/B/GC123ABC/</tt>.
+     * Add spoilers stored locally in <tt>/sdcard/cgeo/GeocachePhotos</tt>. If a cache is named GC123ABC, the
+     * directory will be <tt>/sdcard/cgeo/GeocachePhotos/C/B/GC123ABC/</tt>.
```

Comment:
I think the choice of not using `cgeo` in the path name was because additional data imported from geocaching software could be used by several applications, and did not belong to cgeo.